### ISSUE TITLE
test certs need rotation

### ISSUE THREAD

**Issue by philpennock** [nats-io/nats-server#2497]
Per calendar event created after #1593 it's time to get fresh test certs in the nats-server code-base.

> The nats-server has one-year lifetime certs in the test suite; in part this matches the browser ecosystem moving to a maximum lifetime of one year.  These need to be rotated.
>
> A sample with logs of commands used to generate certs is in https://github.com/nats-io/nats-server/pull/1593 from September 9th 2020.

Looking in `test/configs/certs/` :

```
% find . -name '*.pem' -print | while read -r F; do [[ $F != *key* ]] || continue; printf '%s\t%s\n' "$(openssl x509 -noout -in "$F" -enddate)" "$F"; done | grep 2021
notAfter=Sep  9 11:04:20 2021 GMT	./svid/ca.pem
notAfter=Sep  9 11:34:37 2021 GMT	./svid/client-b.pem
notAfter=Sep  9 11:04:32 2021 GMT	./svid/server.pem
notAfter=Sep  9 11:34:36 2021 GMT	./svid/client-a.pem
notAfter=Sep  9 08:38:28 2021 GMT	./rdns/ca.pem
notAfter=Sep  9 10:15:19 2021 GMT	./rdns/client-c.pem
notAfter=Sep  9 10:09:27 2021 GMT	./rdns/client-b.pem
notAfter=Sep  9 09:14:45 2021 GMT	./rdns/server.pem
notAfter=Sep  9 10:36:25 2021 GMT	./rdns/client-d.pem
notAfter=Sep  9 09:43:09 2021 GMT	./rdns/client-a.pem
```

In case it helps, a pattern to find those ending within N days is:

```sh
DAYS=200
find . -name '*.pem' -print | while read -r F; do
  [[ $F != *key* ]] || continue
  openssl x509 -noout -in "$F" -checkend $((3600*24*DAYS)) >|/dev/null || echo $F
done
```

(Using bash/zsh `[[conditional expressions]]` above to get reliable glob matching)

**Comment by wallyqs** [comment]
Already fixed #2496  ;)

**Comment by philpennock** [comment]
Sorry, that's what I get for getting distracted and not double-checking with a final git update before clicking submit. Thanks!
